### PR TITLE
WIP: Docker php-fpm moved to separate repository

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Need to move php-fpm Dockerfile to separate repository, so it can be automatically build on hub.docker.com
|New feature| Yes
|BC breaks| No
|Fixes issues| No
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: docker/conf/docker-compose-mac.yml.dist [hunk 1]
```diff
@@ -24,12 +24,11 @@ services:
             - smtp-server
 
     php-fpm:
-        build: project-base/docker/php-fpm
+        image: shopsys/framework-php-72-fpm:alpha-1
```
Comment (PetrHeinz) [inline docker/conf/docker-compose-mac.yml.dist]: Why call it `shopsys/framework-php-72-fpm` and not just `shopsys/php-fpm`? Also, I think that the version should not contain just the *alpha1* part, but full version, ie. *7.0.0-alpha1*.

**Comment by PetrHeinz** [comment]
At the moment, we cannot split a new package using the standard job on our CI. There is a possible workaround using manual splitting assuming that the package will not be changed much.

For details see https://github.com/shopsys/shopsys/pull/73#issuecomment-380114874

**Comment by PetrHeinz** [comment]
As there are unresolved comments and a WIP commit was added I'm marking this PR as WIP.

**Comment by PetrHeinz** [comment]
Oh, there is no way to disable the Merge pull request for WIP PRs on GitHub :-1:

**Comment by TomasLudvik** [comment]
This will be part of future US, so I'm closing it.

**Pull request closed without merging**